bin/run.sh

```diff
@@ -22,6 +22,7 @@ SUPPORT_SCHEDULER_LOG_PATH=$BASEPATH/supportScheduler$TIMESTAMPFORMAT.log
 DEVICEVIRTUALLOGSPATH=$BASEPATH/devicevirtual$TIMESTAMPFORMAT.log
 APPSERVICECONFIGURABLELOGSPATH=$BASEPATH/appserviceconfigurable$TIMESTAMPFORMAT.log
 EDGEXLOGSPATH=$BASEPATH/edgex$TIMESTAMPFORMAT.log
+SYSTEMMANAGEMENTCONFIGURABLELOGSPATH=$BASEPATH/systemmanagement$TIMESTAMPFORMAT.log
```

Comment:
Can we update this variable to be consistent with the other `edgex-go` services, maybe `SYSTEMMANAGEMENTLOGSPATH`? and possibly move this up to be with the other `edgex-go` services.?